### PR TITLE
[BRUNCH-73] Bump pg gem

### PR DESCRIPTION
**Pull request by jhan217**
## What did we change?
-added support for pg 1.1.3
-added appraisal cases for both old (0.18.2 per ez-rails) and new (1.1.3) versions of the pg gem
-updated circle to run against a 10.5 postgres db
-updated to version 0.5.0

## Why are we doing this?
to allow dependent applications to upgrade their pg gems

## How was it tested?
- [x] Specs
- [ ] Locally
- [ ] Staging

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
         environment:
           - PGHOST: 127.0.0.1
           - PGUSER: postgres
-      - image: circleci/postgres:9.6
+      - image: circleci/postgres:10.5
```
Comment (jhan217) [inline .circleci/config.yml]: ez-rails Circle config uses the postgres:9.4.14-alpine-postgis-ram image (which has the postgis extension). I don't think it matters too much for this gem, unless we want to keep the db image in sync with what's used in ez-rails, but I thought I'd point it out.

File: Appraisals [hunk 2]
```diff
@@ -2,8 +2,15 @@
 
 appraise "rails-5.1" do
   gem "activerecord", "5.1.6"
+  gem "pg", "0.18.2"
```
Comment (jhan217) [inline Appraisals]: This is the current version used in ez-rails

File: gemfiles/rails_5.1.gemfile [hunk 3]
```diff
@@ -1,7 +1,10 @@
+# frozen_string_literal: true
+
 # This file was generated by Appraisal
 
-source "https://ezcater.jfrog.io/ezcater/api/gems/ezcater-gem-source"
+source "https://rubygems.org"
```
Comment (jhan217) [inline gemfiles/rails_5.1.gemfile]: Not sure what's causing this the generator to defaul to rubygems. Should we change this back to the ezcater source?

File: activerecord-postgres_pub_sub.gemspec [hunk 4]
```diff
@@ -41,7 +41,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "activerecord", ">= 5.1", "< 5.3"
-  spec.add_runtime_dependency "pg", "~> 0.18"
+  spec.add_runtime_dependency "pg", ">= 0.18", "< 1.2"
```
Comment (jhan217) [inline activerecord-postgres_pub_sub.gemspec]: This should allow dependent apps to use anything between 0.18.x to 1.1.x, so I think apps currently using this gem shouldn't be upgrading their pg gems unless the pg gem is being explicitly upgraded in the app.

File: .circleci/config.yml (hunk 1 above)
Comment (tjwp) [reply .circleci/config.yml]: I think this is fine.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,9 @@
 # activerecord-postgres_pub_sub
 
+## v0.5.0
+- Add support for pg 1.1.3
+- Update Circle to run against Postgres 10.5
```
Comment (tjwp) [inline CHANGELOG.md]: I think this line can be dropped from the CHANGELOG as irrelevant to users of the gem.

File: gemfiles/rails_5.1.gemfile (hunk 3 above)
Comment (tjwp) [reply gemfiles/rails_5.1.gemfile]: This is fine. It is probably based on the top-level Gemfile. This is a public gem, so it should not require access to Artifactory gems.

**Pull request merged**